website/views/options/inventory/inventory.jade

```diff
@@ -19,6 +19,11 @@ script(type='text/ng-template', id='partials/options.inventory.equipment.html')
         div
           button.btn.btn-default(type="button", ng-click='dequip("costume");') {{env.t("unequipCostume")}}
           button.btn.btn-default(type="button", ng-click='dequip("petMountBackground");') {{env.t("unequipPetMountBackground")}}
+        li.customize-menu(ng-if='!user.preferences.costume')
+          .well.use-costume-info=env.t('useCostumeInfo1')
+            p
```

Comment:
Let's put this at the bottom. That way, when the more info stuff I isn't being shown, it'll show up in the same place, but when the more info is being shown, it'll be at the bottom.

Also, I learned the other day that you can do: `p: a(ng-click='...`, which when the line is short like this, I think makes sense.